### PR TITLE
Fixed #64 added privacy_policy.html in repo and linked it

### LINKED ISSUE #64: Add a page "Privacy" and provide a link on the footer
We need a page for a privacy policy. Implement a simple site using the same style. Please do the following:
* Add link "Privacy" in the footer on the left hand side
* Implement the following order:  PSLab | FOSSASIA | Privacy
* Also Link "FOSSASIA" to http://fossasia.org

### PR DESCRIPTION
**Pull request by viveksb007**
Fixes #64 

Added privacy_policy.html and linked it in footer of site.

For Testing : https://viveksb007.github.io/pslab.fossasia.org/

screenshot from 2017-08-19 22-58-13

### CODE REVIEW

**Review by akarshan96 — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -418,8 +418,8 @@ <h2 style="">Contact</h2>
 		<hr color="white">
 
     <div>
-			<i class="fa fa-info-circle" aria-hidden="true"></i> PSLab | FOSSASIA
-		</div>
+			<i class="fa fa-info-circle" aria-hidden="true"></i> PSLab | <a href="http://fossasia.org">FOSSASIA</a> <a href="privacy_policy.html">| Privacy</a>
```
Comment (CloudyPadmal) [inline index.html]: Hi @viveksb007 ! What if we exclude the bar line from link and start the anchor tag after it?

Comment (viveksb007) [reply index.html]: yeah the bar line is also anchored to link. I will push a commit to fix it.

**Review by CloudyPadmal — APPROVED** [review]

**Comment by mariobehling** [comment]
@CloudyPadmal @akarshan96 Are you actually testing the site? Top links are not working!

**Review by mariobehling — CHANGES_REQUESTED** [review]
* Top links don't work
screenshot from 2017-08-21 08-58-52

* On top of privacy page, take out "Open Source Pocket Science Lab from FOSSASIA 
PSLAB HARDWARE REPO PSLAB FIRMWARE REPO PSLAB ANDROID REPO" and reduce height of area
* On bottom take out "Contact Us If you have any questions or suggestions about our Privacy Policy, do not hesitate to contact us. dev@fossasia.com"

**Comment by viveksb007** [comment]
@mariobehling I updated the PR description with test-page link yesterday. They approved based on screen cast I guess.
About top links, top bar was made taking main page into consideration where **Getting Started** and **Developers And Mentors** section is.
In privacy policy page there is no such section that why links aren't working.
So I think
- We should remove **Getting Started** and **Developer And Mentor**  from top bar when privacy policy is shown.
- Rest the changes you mentioned

Any more changes or suggestions?

**Comment by viveksb007** [comment]
@mariobehling made the requested changes.

**Comment by mariobehling** [comment]
> We should remove Getting Started and Developer And Mentor from top bar when privacy policy is shown.

Please speed this up and avoid additional changes. Just stay with the issue description. Top bar should stay the same as on main page.

**Comment by viveksb007** [comment]
@mariobehling all the links in top bar works now. No additional change in top bar.

File: privacy_policy.html [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+                                                                                style="cursor: pointer;"></a>
+            <ul class="nav navbar-nav navbar-right">
+                <li id="fossasia-link">
+                    <a href="http://fossasia.org" target="_blank">FOSSASIA</a>
+                </li>
+                <li>
+                    <a href="index.html" title="Home">Home</a>
+                </li>
+                <li>
+                    <a href="#features" title="Features">Features</a>
```
Comment (CloudyPadmal) [inline privacy_policy.html]: Hi @viveksb007 ! I guess this link needs to be changed right? As this points to a div. Does it work in your pc?

Comment (viveksb007) [reply privacy_policy.html]: Sorry my bad. Missed one, it should be `index.html#features`

**Comment by mariobehling** [comment]
Please rename "privacy_policy.html" to "privacy.html"

**Comment by viveksb007** [comment]
@mariobehling renamed **privacy_policy.html** to **privacy.html**

**Review by mariobehling — APPROVED** [review]

**Comment by mariobehling** [comment]
Thanks!

**Pull request merged**